group/group.go

```diff
@@ -30,6 +30,7 @@ type Group interface {
 	RandomScalar(io.Reader) Scalar
 	HashToElement(data, dst []byte) Element
 	HashToScalar(data, dst []byte) Scalar
+	Len() Length
```

Comment:
```suggestion
	Length() Length
```

Alternatively, can we just inline the Length struct here? I don't see why this layer of indirection is useful.

Reply:
The `Length` function is needed b/c this is an interface, right?

Reply:
Sorry, I mean, add three functions on this interface: one for element length, one for compressed element length, and one for scalar length. I don't see why we need a function on the interface that just returns this struct. Alternatively, you could generalize this a bit and call it `GroupParams`, returning a struct called `GroupParams`, that has these fields in it. Then you could add other group-related parameters, sort of like [what Go does with curves](https://golang.org/pkg/crypto/elliptic/#CurveParams).

Reply:
Renaming done.
Luke: Yes, we need this function in the interface.